src/components/players/details/totalStats/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import Spinner from 'components/_shared/spinner';
 import Table from 'components/_shared/table';
-import Title from 'components/_shared/title';
+import Title from 'components/_shared/title'
```

Comment:
Missing semi-colon

Reply:
noted, will make the changes when i merge the code over.